server/service/IndividualServicesService.js

```diff
@@ -54,7 +54,7 @@ exports.bequeathYourDataAndDie = function (body, user, originator, xCorrelator,
   return new Promise(async function (resolve, reject) {
     try {
 
-        const FcportValue = 'PromptForBequeathingDataCausesTransferOfListOfApplications';
+      const fcportValue = 'PromptForBequeathingDataCausesTransferOfListOfApplications';
```

Comment:
If you make this a constant in SoftwareUpgrade file, you can use it here and you don't have to send it over as parameter.

Reply:
removed the fcportvalue done